src/ios/CDVWKInAppBrowser.m

```diff
@@ -1097,7 +1097,8 @@ - (void)close
 - (void)navigateTo:(NSURL*)url
 {
     if ([url.scheme isEqualToString:@"file"]) {
-        [self.webView loadFileURL:url allowingReadAccessToURL:url];
+        NSURL* directoryURL = [url URLByDeletingLastPathComponent];
+        [self.webView loadFileURL:url allowingReadAccessToURL:directoryURL];
```

Comment:
Would like allow reading the local `cordova.js` file? If so that could be **big** trouble...

Reply:
This does not change current logic anyhow. This only enables later ios versions to load static files from `url`.

Use case is - we put static content in our apps and then app reads this static content via built in webserver / directly (css files/images etc).

Reply:
This fix is really a lifesaver for accessing local resources.
Fixes "The operation couldn’t be completed. (kCFErrorDomainCFNetwork error 1.)" error for me, and saved my clients apps!
Any chance this gets committed any time soon?